package.json

```diff
@@ -57,7 +57,8 @@
         "typescript": "^5.5.4",
         "typescript-eslint": "^8.4.0",
         "unist-builder": "2",
-        "unist-util-visit": "^2"
+        "unist-util-visit": "^2",
+        "ilib-loctool-pendo-md": "file:."
```

Comment:
What?

Reply:
Loctool was complaining that it can't find the plugin https://github.com/wadimw/ilib-loctool-pendo-md/pull/3#discussion_r1776872388

Reply:
The project.json could just give the path directly to the pendo-md root, but I guess this is fine.